iam-keycloak-migration.sh

```diff
@@ -407,7 +407,7 @@ function inspectIdpConnections {
 }
 
 function getKeycloakGroup() {
-  keycloakGroups="$(curl -ks "https://$keycloakUrl/admin/realms/$keycloakCloudPakRealm/ui-ext/groups?search=$1&exact=true" \
+  keycloakGroups="$(curl -ks --get "https://$keycloakUrl/admin/realms/$keycloakCloudPakRealm/groups" --data-urlencode "search=$1&exact=true" \
```

Comment:
I think that would urlencode the "&" which is not what we want.

```suggestion
  keycloakGroups="$(curl -ks --get "https://$keycloakUrl/admin/realms/$keycloakCloudPakRealm/groups" --data-urlencode "search=$1" --data-urlencode "exact=true" \
```